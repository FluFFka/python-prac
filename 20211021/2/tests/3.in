:fan q exec('import'+f'%c'%32+'time\na="\|/-"\nfor'+f'%c'%32+'i'+f'%c'%32+'in'+f'%c'%32+'range(q+3-q%4):\n'+f'%c'%32+f'%c'%32+f'%c'%32+f'%c'%32+'print("\\r"+a[(i+1)%4]+"\a",end="")\n'+f'%c'%32+f'%c'%32+f'%c'%32+f'%c'%32+'time.sleep(0.03)\nprint(">lol\vlol\vlol\vlol<-")')
fan 10
fan 5
fan 1
fan 4
quit "{}\r{}_strings"
