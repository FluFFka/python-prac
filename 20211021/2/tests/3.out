|/-\|/-\|/->lollollollol<-
None
|/-\|/->lollollollol<-
None
|/->lollollollol<-
None
|/-\|/->lollollollol<-
None
26_strings
